README.md

```diff
@@ -21,7 +21,7 @@ func main() {
 	module, _ := wazero.NewRuntime().NewModuleFromSource(source)
 
 	// Discover 7! is 5040
-	fmt.Println(module.Function("fac").Call(context.Background(), 7))
+	fmt.Println(module.ExportedFunction("fac").Call(nil, 7))
```

Comment:
Note: this prefix is to retain access to `Memory()` without necessarily exporting it. Later changes may decide to do similar with functions and globals, but that would be too much here.